src/3d/qgs3dmapsettings.h

```diff
@@ -200,6 +201,35 @@ class _3D_EXPORT Qgs3DMapSettings : public QObject
     //! Returns terrain generator. It takes care of producing terrain tiles from the input data.
     QgsTerrainGenerator *terrainGenerator() const { return mTerrainGenerator.get(); }
 
+    /**
+     * Sets whether terrain shading is enabled.
+     * \sa isTerrainShadingEnabled()
```

Comment:
I don't think sipify can handle "sa" - can you change this to "\see" instead?

Reply:
updated in bcf8b0d